ecl/hthor/hthor.ipp

```diff
@@ -171,7 +170,7 @@ static bool verifyFormatCrc(unsigned helperCrc, IDistributedFile * df, char cons
 
 static bool verifyFormatCrcSuper(unsigned helperCrc, IDistributedFile * df, bool isIndex, bool fail)
 {
-    if(!df) return true;
+    if(!df || (helperCrc == 0)) return true;
```

Comment:
was this a bug before? If helperCrc was 0 it would check and fail?

Reply:
Probably why the helper returned TDRnocrccheck.  I think that's why I needed to add this.